index.html

```diff
@@ -18,11 +17,41 @@
 			</form>
 		</div>
 		<br>
-	</div>
-	
+	</div>	
 	<div id="page">
 
 	</div>
+	<script type="text/javascript">
+		var searchDiv = document.getElementById("search");
```

Comment:
Why get the div and then get the from from the div?

Reply:
I figured I would know I have the right form selected in case other markup gets added to the page.  

Other options would have been:

document.getElementsByTagName("form")[0]
document.forms[0]
document.getElementById("search").childNodes[0]

I ran a benchmark on it though and it looks like fastest to just use getElementsByTagName("form") on the whole document.  I'm not sure how true that stays as you scale up the amount of markup on a page.

https://www.measurethat.net/Benchmarks/Show/5603/0/getelementbyidgetelementsbyclassname-vs-getelementsbycl